packages/exporter-trace-otlp-http/src/platform/browser/util.ts

```diff
@@ -52,7 +52,8 @@ export function sendWithXhr(
   url: string,
   headers: Record<string, string>,
   onSuccess: () => void,
-  onError: (error: otlpTypes.OTLPExporterError) => void
+  onError: (error: otlpTypes.OTLPExporterError) => void,
+  exporterTimeout: number
```

Comment:
usually callbacks are last but not sure how often this API is used and if it is exported to public. So feel free to keep as is.

Reply:
it is not public